### ISSUE TITLE
Freshly installed Laravel 5.1, added Ticketit v0.2.4, getting error when migrating: "Undefined variable: migration_arr"

### ISSUE THREAD

**Issue by jrequiroso** [thekordy/ticketit#173]
The offending code is here:

```
   $migration_arr = array(); //add this line to prevent the error below from happening.
   foreach ($migrations as $migration_parent) { // Count active package migrations
        $migration_arr [] = $migration_parent->migration;
    }

    foreach ($tables as $table) {
        if (!in_array($table, $migration_arr)) { //this is not detecting that $migration_arr is empty
            $inactiveMigrations [] = $table;
        }
    }
```

**Comment by tmcdon89** [comment]
Thanks for the report.

**Comment by tmcdon89** [comment]
Fixed in f21a7fd
Thanks again for the contribution!
